NetCord/PartialGuildUserExtensions.cs

```diff
@@ -42,6 +67,12 @@ public static Permissions GetChannelPermissions(this PartialGuildUser user, Guil
         return user.GetChannelPermissionsCore(guildPermissions, guild.Channels[channelId]);
     }
 
+    /// <summary>
+    /// Returns a <see cref="IGuildChannel"/>-specific <see cref="Permissions"/> object belonging to the <see cref="PartialGuildUser"/> by acquiring it from the specificied <paramref name="guildPermissions"/>.
+    /// </summary>
+    /// <param name="user"> The <see cref="PartialGuildUser"/> to acquire permissions for. </param>
+    /// <param name="guildPermissions"> The <see cref="Permissions"/> object to acquire permissions from. </param>
+    /// <param name="channel"> The <see cref="IGuildChannel"/> to acquire the permissions for. </param>
```

Comment:
That's exactly what the method does no?

Reply:
Not really, it uses the already computed guild wide permissions so that you can efficiently calculate a channel wise ones. What I am not sure is if the doc makes sense.

Reply:
also there are leading and trailing spaces here

Reply:
Fixed the spacing across the file, still confused about what you mean though. You're passing a permissions object, and its extracting channel permissions from them, isn't it?

Reply:
It uses guild wide permissions to calculate channel permissions. What I mean is that "Permissions object to acquire permissions from" seems a bit odd to me. But well, it's technically correct. Also maybe the docs should explicitly state that they have to be guild wide permissions.

Reply:
I don't think that's necessary given that the parameter's name has guild in it.